src/Control/LDAPDebugController.php

```diff
@@ -18,6 +18,9 @@
  * Class LDAPDebugController
  *
  * This controller is used to debug the LDAP connection.
+ *
+ * @template T of SiteTree
+ * @extends ContentController<T>
```

Comment:
```suggestion
 * @extends ContentController<SiteTree>
```
You'll also need to import SiteTree

I haven't consistently checked that things like SiteTree, DataObject etc have been imported for use in docblocks, you should probably spin off a new card to validate docblock imports

Reply:
I had this extensible so people can subclass it with their own type - but that's probably unlikely enough that making the suggested change is fine.
Done

Reply:
> I haven't consistently checked that things like SiteTree, DataObject etc have been imported for use in docblocks, you should probably spin off a new card to validate docblock imports

I've added this as an AC for https://github.com/silverstripe/.github/issues/181